packages/babel-helpers/src/helpers/isNativeFunction.ts

```diff
@@ -1,10 +1,10 @@
 /* @minVersion 7.0.0-beta.0 */
 
-export default function _isNativeFunction(fn) {
+export default function _isNativeFunction(fn: unknown): fn is Function {
   // Note: This function returns "true" for core-js functions.
   try {
-    return Function.toString.call(fn).indexOf("[native code]") !== -1;
-  } catch (e) {
+    return Function.toString.call(fn).includes("[native code]");
```

Comment:
Can you undo this change? Because `includes` are not supported in older engines.

Reply:
yes I will now

the reason I changed it is because it was giving me an error on ci pipline